Update CONTRIBUTING.md about Homebrew/cask-versions repo (#173970)
Homebrew/homebrew-cask: CONTRIBUTING.md

## Adding a Cask

> Notice an application that's not in Homebrew Cask yet? Make sure it's not yet in this repo or [homebrew/core](https://github.com/Homebrew/homebrew-core) (can be searched with `brew search`). Mac App Store apps can't be installed via Homebrew Cask, but check out [mas](https://github.com/mas-cli/mas) for an alternative.

Reviewer: ```suggestion
Notice an application that's not in Homebrew Cask yet? Make sure it's also not in [homebrew/core](https://github.com/Homebrew/homebrew-core) (can be searched with `brew search`). Mac App Store apps can't be installed via Homebrew Cask, but check out [mas](https://github.com/mas-cli/mas) for an alternative.
```
